buildkitd/Earthfile

```diff
@@ -6,12 +6,12 @@ buildkitd:
     IF [ "$BUILDKIT_PROJECT" != "" ]
         IF case "$BUILDKIT_PROJECT" in "../*") true;; "*") false;; esac
             # Assuming this is coming from the main Earthly Earthfile.
-            ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:00025901bf6b30a374e68eecf9fab9c28554bd25+build
+            ARG BUILDKIT_BASE_IMAGE=../$BUILDKIT_PROJECT+build
```

Comment:
looks like the update-buildkit target replaced these accidentally in https://github.com/earthly/earthly/commit/70decc75441ae9c3b086361d4ae37a6700e68fb4#diff-e3b996d9f9912a0d82d4146e5f317bc32c39f22a3fcd0e8cdfb070e7a2a7a2e4

reverting this.

Reply:
and I just checked that `earthly +update-buildkit` no longer updates these two default arg values.